server/asset/eth/client_harness_test.go

```diff
@@ -0,0 +1,61 @@
+// +build harness
```

Comment:
Having `client` in the filename is a little confusing, since this is the server backend.

Reply:
Renamed most `client` to `rpcclient` for clarity. Better that way?